### ISSUE TITLE
[Bug]: touch-action:none doesn't stop overscroll that hides navigation bar

### ISSUE THREAD

**Issue by smellymoo** [mozilla-mobile/fenix#22458]
### Steps to reproduce

Pointer-events using either **touch-action:none** or **overscroll-behavior:none** on a full-screen element with **overflow:hidden** (so no scrolling).

1) quickly touch element and slide up.

### Expected behaviour

no gestures, only the element should receive pointer-events.

### Actual behaviour

over-scroll happens and allows you to scroll down and hide the navigation bar.

### Device name

Galaxy Xcover pro

### Android version

Android 11

### Firefox release type

Firefox

### Firefox version

93.2.0

### Additional information

I have also tried using preventDefault(). But by my understanding gestures on Pointer-events will bypass that.

Chromium does what is expected.

**Comment by Mugurell** [comment]
Thank you for the report!
Can you please confirm if this issue also reproduces on `geckoview_example.apk` from here https://firefox-ci-tc.services.mozilla.com/tasks/index/gecko.v2.mozilla-central.latest.mobile/android-aarch64-debug ?

**Comment by smellymoo** [comment]
I installed geckoview_example.apk but it didn't add any app?

**Comment by smellymoo** [comment]
yes it does the same problem. Navbar is hiding even with touch-action:none or overscroll-behavior:none enabled.

**Comment by Mugurell** [comment]
Thank you!
That is a different app with little functionality added on top GeckoView which allows testing it in isolation.
If the issue happens there also then this issue will have to be investigated on GeckoView's side so I'll move the ticket to Bugzilla. Please follow the progress there.

**Comment by Mugurell** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1741419

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).

**Comment by Mugurell** [comment]
@smellymoo  Can you share a simple page to test this?
A video of the issue you're seeing would also be great!
